pkg/local_object_storage/peapod/get.go

```diff
@@ -0,0 +1,42 @@
+package peapod
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util/slice"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+// Get reads data from the underlying database by the given object address.
+// Returns apistatus.ErrObjectNotFound if object is missing in the Peapod.
+func (x *Peapod) Get(addr oid.Address) ([]byte, error) {
+	var data []byte
+
+	err := x.bolt.View(func(tx *bbolt.Tx) error {
+		bktRoot := tx.Bucket(rootBucket)
+		if bktRoot == nil {
+			return fmt.Errorf("%w: missing root bucket", apistatus.ErrObjectNotFound)
```

Comment:
We usually use `fmt.Errorf("some text: %w", err)` pattern. In this function exists two different patterns for errors, with `%w` in the end and in the beginning.

Is it an exceptional situation to change this format here?

Reply:
usually reason goes after colon, and here `missing root bucket` is a particular reason of `object not found` (there may be other reasons). Here we could just `return apistatus.ErrObjectNotFound`, but additional context is good.

u may encounter similar cases in the code base